docs/reference/glossary.md

```diff
@@ -149,6 +149,10 @@ The log is comprised of an ordered sequence of records written to persistent sto
 
 - [Partitions](/components/zeebe/technical-concepts/partitions.md#partition-data-layout)
 
+### Management cluster
```

Comment:
Lets call it "Web Modeler and Console clusters" we are not planning to use Management cluster for now.

Reply:
I removed this term